setup.py

```diff
@@ -182,7 +182,7 @@ def check_for_openmp():
 
         "extras_require": {':python_version=="2.6"': ['argparse>=1.2.1'],
                            'docs': ['sphinx', 'sphinxcontrib-autoprogram'],
-                           'tests': ['nose >= 1.0']},
+                           'tests': ['nose >= 1.0', 'pysam']},
```

Comment:
On Wed, May 20, 2015 at 07:53:43AM -0700, Michael R. Crusoe wrote:
> > @@ -182,7 +182,7 @@ def check_for_openmp():
> >  
> >          "extras_require": {':python_version=="2.6"': ['argparse>=1.2.1'],
> >                             'docs': ['sphinx', 'sphinxcontrib-autoprogram'],
> > -                           'tests': ['nose >= 1.0']},
> > +                           'tests': ['nose >= 1.0', 'pysam']},
> 
> Agreed, but it fails tests due to the import otherwise. 

Yes, we should either skip the test in that case or find a lighterweight
dep, I think.